### ISSUE TITLE
[BUG] random evos

### ISSUE THREAD

**Issue by DeadByte9** [rotobash/pokemon-ngc-rando#18]
im trying to use the randomized evo option but everything is evolving into their regular evo forms even the logs says that the evos are random when they aren't

**Comment by rotobash** [comment]
Hi there!

Thanks for reporting this, I'll look into it as soon as I can. In the meantime, can you tell me if you were using the latest version and possibly what settings you used?

Cheers

**Comment by DeadByte9** [comment]
yes i was using the latest version 1.1.5.0 I did evolutions randomize fix impossible easier evolutions similar strength and same type which is everything lol

**Comment by rotobash** [comment]
Hi there, 

First of all, sorry for the long delay in responding to this. I haven't forgotten or abandoned this, it's just that December was a hectic month for me. 

Disregarding the excuses, I think I know what has happened here and where to look to solve this issue. Something got messed up when adding Colosseum support, so while Colosseum works, XD does not anymore. Unfortunately, it requires painstakingly comparing old commits and seeing what specific change caused this. I hope to sit down and figure out exactly what happened soon.

In the meantime, if you'd still like to use the randomization for XD, version 1.0.6-beta still works for XD. It's not an ideal solution but it'll have to do for now until I can fix the issue.

Thanks for reporting this and for your patience!

**Comment by rotobash** [comment]
I believe I have solved this issue now. When adding Colosseum support I crossed the wires somewhere and it was reading evolution data wrong. You would probably get normal pokemon to evolve into, but the conditions were randomized to garbage. I'm actually surprised it didn't crash the game to be honest. Just goes to show how well these games are programmed.

Anyway, I've made a new release. If you have some time give it a spin and let me know if you encounter anything else. Cheers.
